extensions/interactions/GraphInput/GraphInput.js

```diff
@@ -583,6 +583,86 @@ oppia.directive('graphViz', function() {
 
 oppia.factory('graphUtilsService', [function() {
   return {
+    GRAPH_ADJACENCY_MODE: {
+      DIRECTED: 'directed',
+      INVERTED: 'inverted',
+      UNDIRECTED: 'undirected'
+    },
+    DFS_STATUS: {
+      VISITED: 'visited',
+      UNVISITED: 'unvisited',
+      STILL_VISITING: 'still visiting'
+    },
+    /**
+     * @param {object} graph A graph object.
```

Comment:
I would put a colon after the first 'graph' to make this easier to read. Likewise below.